test/app.test.js

```diff
@@ -1,33 +1,65 @@
 var AppTester = vumigo.AppTester;
 var fs = require('fs');
+var onafixtures = require('./ona.fixtures');
 
 
 describe("app", function() {
     describe("GoApp", function() {
         var app;
         var tester;
+        var now;
+        var completed_answers = {
+            Facility_Code_Entry: '111111',
+            First_Name_Entry: 'First_Name_Entry',
+            ID_Type_Entry: 'ID_Type_Entry',
+            Last_Name_Entry: 'Last_Name_Entry',
+            Locality_Entry: 'Locality_Entry',
+            Locality_Entry_Other: 'Locality_Entry_Other',
+            MSISDN_Entry: 'MSISDN_Entry',
+            date_of_birth: '19800102',
+            Patient_Abroad_Entry: 'Patient_Abroad_Entry',
+            SA_ID_Entry: '8905100273087',
+            No_SA_ID_Gender_Entry: 'gender',
+            Landmark_Entry: 'Landmark_Entry',
+            Landmark_Entry_Description: 'Landmark_Entry_Description'
+        };
 
         beforeEach(function() {
-            app = new go.app.GoApp();
-
-            tester = new AppTester(app);
-
-            tester
-                .setup.config.app({
-                    name: 'test_app',
-                    api_endpoint: 'http://www.example.org/api/v1/',
-                    facilities: JSON.parse(
-                        fs.readFileSync(
-                            "src/lookups/facilities.json", "utf8")),
-                    landmarks: JSON.parse(
-                        fs.readFileSync(
-                            "src/lookups/landmarks.json", "utf8"))
-
-                })
-                .setup(function(api) {
-                    fixtures().forEach(api.http.fixtures.add);
+            return make_im()
+                .then(function(im) {
+                    now = moment();
+                    go.utils.now = function () {
+                        return moment("2015-01-01T00:00:00+00:00");
+                    };
+                    app = new go.app.GoApp();
+                    tester = new AppTester(app);
+
+                    tester
+                        .setup.config.app({
+                            name: 'test_app',
+                            ona: {
+                                id: '1',
+                                username: 'root',
+                                password: 'root',
+                                url: 'http://ona.io/api/v1/'
+                            },
+                            api_endpoint: 'http://www.example.org/api/v1/',
+                            landmarks: JSON.parse(
+                                fs.readFileSync(
+                                    "src/lookups/landmarks.json", "utf8"))
+                        })
+                        .setup(function(api) {
+                            fixtures().forEach(api.http.fixtures.add);
+                            onafixtures.store.forEach(function (fixture) {
+                                api.http.fixtures.add(fixture);
+                            });
```

Comment:
All these changes are to make the app aware of Ona and to load Ona fixtures as HTTP fixtures which we can introspect in the tests.